### PR TITLE
Camera: upload events through API endpoint

### LINKED ISSUE #112: Send monitoring starts/ends events

### PR DESCRIPTION
**Pull request by kantoniak**
* Update old API definition with generated code
* Upload start/end events
* Upload enter/leave events in separate requests

Timestamps are currently either `datetime`s or `float`s. I will clean it up in a follow-up PR.
Closes #112.

### CODE REVIEW

File: camera/rpi-files/uploader.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            RuntimeError: on upload failure
+
+        Returns:
+            HTTP response object
+        """
+        camera_pk = str(self._device_id)
+        try:
+            self._api.cameras_events_create(camera_pk, event)
+        except ApiException as ex:
+            msg = 'Could not upload event: {0}'.format(ex)
```
Comment (avan1235) [inline camera/rpi-files/uploader.py:78]: rremember of fstrings, it jsut get's simpler in a way f'Could not upload event: {ex}'

Comment (kantoniak) [reply camera/rpi-files/uploader.py:78]: `fstring`s are forbidden per used style guide. See [wemake-python-styleguide#32](https://github.com/wemake-services/wemake-python-styleguide/issues/32) for discussion.

File: camera/rpi-files/main.py [hunk 2]
```diff
@@ -125,11 +146,27 @@ def _start(self):
 
             # Count iterations for demo purposes
             iteration += 1
-            if iteration == 10:
+            if iteration == 6:
```
[the remark this answers is not in the document]
Comment (kantoniak) [reply camera/rpi-files/main.py]: Removed completely.

File: camera/rpi-files/uploader.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+from data_batcher import Batch
+from events import EventType, create_event
+from openapi_client import ApiClient, CameraEvent, CamerasApi, Configuration
+from openapi_client.rest import ApiException
+
+
+class Uploader(object):
+    """Uploads events to API endpoint."""
+
```
[the remark this answers is not in the document]
Comment (kantoniak) [reply camera/rpi-files/uploader.py]: * Merged `__init__` and `start`
* Renamed `Uploader` to `ApiConnector`

**Pull request merged**